cdc/kv/client.go

```diff
@@ -234,6 +234,7 @@ type CDCClient struct {
 	}
 	ingressCommitTs   model.Ts
 	ingressResolvedTs model.Ts
+	filterLoop        bool
```

Comment:
how about add a comment about this field to indicate it's usage ?

Reply:
sure